app/views/team-members/team-member-invite.njk

```diff
@@ -24,61 +24,158 @@ Invite a new team member - GOV.UK Pay
+      {{ govukInput({
           label: {
-            classes: "govuk-label--s"
+            text: "Email address"
           },
-          hint: {
-            html: "View transactions<br/>Refund payments</br/>Manage settings"
+          id: "invitee-email",
+          name: "invitee-email",
+          classes: "govuk-!-width-two-thirds",
+          type: "email"
+        })
+      }}
+
+      {{ govukRadios({
+        idPrefix: "role-input",
+        name: "role-input",
+        fieldset: {
+          legend: {
+            text: "Permission level",
+            classes: "govuk-label--s"
           }
         },
-        {
-          value: viewAndRefund.id,
-          text: "View and refund",
-          label: {
-            classes: "govuk-label--s"
+        items: [
+          {
+            value: admin.id,
+            text: "Administrator",
+            label: {
+              classes: "govuk-label--s"
+            },
+            hint: {
+              html: "View transactions<br>
+              Refund payments<br>
+              Take telephone payments<br>
+              Manage settings"
+            }
+          },
+          {
+            value: viewAndRefund.id,
+            text: "View and refund",
+            label: {
+              classes: "govuk-label--s"
+            },
+            hint: {
+              html: "View transactions<br>
+              Refund payments<br>
+              Cannot take telephone payments<br>
+              Cannot manage settings"
+            }
           },
-          hint: {
-            html: "View transactions<br/>Refund payments</br/>Cannot manage settings"
+          {
+            value: view.id,
+            text: "View only",
+            label: {
+              classes: "govuk-label--s"
+            },
+            checked: true,
+            hint: {
+              html: "View transactions<br>
+              Cannot refund payments<br>
+              Cannot take telephone payments<br>
+              Cannot manage settings"
+            }
+          },
+          {
+            value: viewAndInitiateMoto.id,
+            text: "View and take telephone payments",
+            label: {
+              classes: "govuk-label--s"
+            },
+            hint: {
+              html: "View transactions<br>
+              Cannot refund payments<br>
+              Take telephone payments<br>
+              Cannot manage settings"
+            }
+          },
+          {
+            value: viewRefundAndInitiateMoto.id,
+            text: "View, refund and take telephone payments",
+            label: {
+              classes: "govuk-label--s"
+            },
+            hint: {
+              html: "View transactions<br>
+              Refund payments<br>
+              Take telephone payments<br>
+              Cannot manage settings"
+            }
           }
-        },
-        {
-          value: view.id,
-          text: "View only",
+        ]
+      }) }}
+    {% else %}
```

Comment:
Is it possible to extract everything under this `else` block, seeing as its common to both cases (where service has agent initiated moto enabled and where it hasn't)?

Reply:
Not everything because the hints are different for the different roles depending on whether the service has the flag or not. But there’s no need for the email field code to be duplicated so I’ll fix that.

Reply:
OK, done that.